README.md

```diff
@@ -367,6 +367,21 @@ Feel free to clone and modify this project. Pull requests are welcome.<br>You ca
 
 We follow the [Conventional Commits](https://www.conventionalcommits.org/) specification for all commits. Please make sure your commit messages adhere to this standard.
 
+### TroubleShooting
```

Comment:
Hey, sorry for the little nitpick, but "troubleshooting" is one word, so the capital isn't necessary on the 's'.

I would suggest changing this just a little bit:
When attempting to launch a local network, if you encounter an error message similar to the following:
...

It indicates a change in the `NodeRegistry` struct, such that it is no longer compatible with a previously serialized representation.

Delete the node registry file on your platform:
C:\Users\<username>\AppData\Roaming\safe\node_registry.json
~/.local/share/safe/node_registry.json